ako-gateway-api/status/gateway_status.go

```diff
@@ -148,6 +148,10 @@ func (o *gateway) Update(key string, option status.StatusOptions) {
 	}
 
 	gatewaystatus := akogatewayapiobjects.GatewayApiLister().GetGatewayToGatewayStatusMapping(gw.Namespace + "/" + gw.Name)
+	if gatewaystatus == nil {
```

Comment:
@ashishnayak96: This is being addressed as part of this PR: https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1593/files#diff-6ab3147012ebebb189722a3e384914e6570b9aacf9757a0ac17946cfac1ae952

Reply:
removed